src/routes/SearchRoute/SearchRoute.js

```diff
@@ -34,12 +34,17 @@ const SearchRoute = ({ children }) => {
   } = useContext(AuthoritiesSearchContext);
   const [, setSelectedAuthorityRecordContext] = useContext(SelectedAuthorityRecordContext);
 
+  const sortableColumns = [
+    searchResultListColumns.AUTH_REF_TYPE,
+    searchResultListColumns.HEADING_REF,
+    searchResultListColumns.HEADING_TYPE,
+  ];
   const {
     sortOrder,
     sortedColumn,
     onChangeSortOption,
     onHeaderClick,
-  } = useSortColumnManager({ sortableColumns: sortableSearchResultListColumns });
+  } = useSortColumnManager(sortableColumns);
```

Comment:
Should be an object with `sortableColumns` properties here

Reply:
as per our discussion, shouldn't it be an array?

Reply:
passed sortableColumns as an object parameter